src/Fields/StringField.php

```diff
@@ -19,11 +21,14 @@ public function inferProperties($val, $lenient = false)
      *
      * @return string
      *
-     * @throws \frictionlessdata\tableschema\Exceptions\FieldValidationException;
+     * @throws FieldValidationException;
      */
     protected function validateCastValue($val)
     {
         try {
+            if (is_object($val)) {
+                throw $this->getValidationException('Could not cast object to string', $val);
+            }
             $val = (string) $val;
         } catch (\Exception $e) {
```

Comment:
The latest set of changes in `frictionlessdata:master` drop support for PHP 5.6.

With support for 5.6 dropped, I believe a better correction for PHP 7.4 will the following.
```suggestion
        try {
            $val = (string) $val;
        } catch (\Throwable $e) {
```

Reply:
Done.